README.md

```diff
@@ -0,0 +1,29 @@
+# LiligoS3Dash
+
+## Init Git Submodules
+
+We use submodules to manage components. To download the deps, follow the instructions below:
+
+```
+git submodules init
+git submodules update
+```
+
+## Environment setup
+
+Follow [ESP32 installation guide](https://docs.espressif.com/projects/esp-idf/en/v4.3.1/esp32/get-started/index.html#installation-step-by-step) to setup your local environment. 
```

Comment:
The project now requires v5 or above. So we should update this link to ESP-IDF 5